packages/cli/src/commands/transfer.ts

```diff
@@ -0,0 +1,63 @@
+import { ZWeb3 } from 'zos-lib';
+import transfer from '../scripts/transfer';
+import { promptIfNeeded, networksList, InquirerQuestions } from '../prompts/prompt';
+import { isValidUnit } from '../utils/units';
+import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesInitializer';
+
+const name: string = 'transfer';
+const signature: string = name;
+const description: string = 'send funds to a given address';
+
+const register: (program: any) => any = (program) => program
+  .command(signature, undefined, { noHelp: true })
+  .usage('[options]')
+  .description(description)
+  .option('--to <to>', 'specify recipient address')
+  .option('--value <value>', 'the amount of ether units to be transferred')
+  .option('--unit <unit>', 'unit name. If none is given, \'ether\' will be used. To see supported units, go to: https://web3js.readthedocs.io/en/1.0/web3-utils.html#id85')
+  .withNetworkOptions()
+  .withNonInteractiveOption()
+  .action(action);
+
+async function action(options: any): Promise<void> {
+  const { network: networkInOpts, unit, to, value, from, interactive } = options;
+  const configOpts = { network: networkInOpts, from };
+  const configProps = getCommandProps();
+  const promptedConfig = await promptIfNeeded({ opts: configOpts, props: configProps }, interactive);
+  const { txParams } = await ConfigVariablesInitializer.initNetworkConfiguration(promptedConfig, true);
+
+  const transferOpts = { from, to, value };
+  const transferProps = getCommandProps(await ZWeb3.accounts(), unit);
+  const promptedTransfer = await promptIfNeeded({ opts: transferOpts, props: transferProps }, interactive);
+
+  await transfer({ ...promptedTransfer, unit, txParams });
+
+  if (!options.dontExitProcess && process.env.NODE_ENV !== 'test') process.exit(0);
+}
+
+function getCommandProps(accounts: string[] = [], unit: string = 'ether'): InquirerQuestions {
+  return {
+    ...networksList('network', 'list'),
+    from: {
+      type: 'list',
+      message: 'Choose the account to send transactions from',
+      choices: accounts.map((account, index) => ({
+        name: `(${index}) ${account}`,
+        value: account
+      }))
+    },
+    to: {
+      type: 'input',
+      message: 'Enter a receiver account'
+    },
+    value: {
+      type: 'input',
+      message: 'Enter an amount to transfer',
```

Comment:
Can we include the `unit` as part of the message, so we remind the user right here?

Reply:
We will be reminding them the unit while they type the amount value (with the `transformer` function)